### DISCUSSION TITLE
[SOLVED] [Feature Request] 9 Panel Techno

### DISCUSSION THREAD

**Discussion by Casuallynoted** [TeamRizu/OutFox#370] [03 - Feature Requests]
Hello!

I wanted to drop in a suggestion for 9-panel support for techno mode. While the original game only supported 8 arrows, the pad has the ability to support 9-panel play in theory. It might be neat to have this capability for people who want even more challenge, or just want to do it because why not!

**Comment by SHRMP0** [comment]
I believe this was already implemented on [Alpha 4.9.8](https://projectmoon.dance/news/project-outfox-53-alpha-498-released). If you're using an older build, please update.

**Reply by Casuallynoted** [reply]
So it was! I was being an idiot and didn't realise you could go even lower on the list of difficulty modes to get to 9 panel! 😆 I'm sorry about that! Thank you guys so much for adding this!

**Reply by Scraticus** [reply]
no problems, it was an obscure mode as it was for Techno, i've only known of it working on the 'style change' charts on the singapore cabs, but it was logical to have it as techno did have a 9 panel pad.
